### DISCUSSION TITLE
Manage Visible Fields not Remembering State

### DISCUSSION THREAD

**Discussion by HobbesKo** [stacks-task-manager/stacks2#2] [Bugs]
I can check and uncheck Main fields and Footer fields, and the changes are reflected in the app.
However, on restart of Stacks 2 my edits are ignored and all fields are checked and visible.

Stacks 2 Version 1.0.0-a51

**Comment by synapse** [comment]
Hey @HobbesKo, should be fixed in [v1.0.0-a52](https://github.com/stacks-task-manager/stacks2/releases/tag/v1.0.0-a52)

**Comment by HobbesKo** [comment]
Yep, all fixed. Much thanks.
